src/Microsoft.OData.Core/ODataConstants.cs

```diff
@@ -218,7 +218,7 @@ internal static class ODataInternalConstants
         /// <summary>
         /// The default size used for buffering output data.
         /// </summary>
-        internal const int DefaultOutputBufferSize = 84000;
+        internal const int DefaultOutputBufferSize = 16 * 1024;

```

Comment:
@habbes Do we have any data to suggest that 16,384 (16*1024) is the sweet spot compared to 84,000? For example, I'd be curious if the higher number is more ideal in scenarios where payloads are much bigger...

Reply:
I restored the default buffer size to 84,000 but added a `BufferSize` property to the `ODataMessageWriterSettings` to configure this value.